backend/typeDefs.js

```diff
@@ -0,0 +1,39 @@
+const { gql } = require("apollo-server");
+
+
+
+const typeDefs = `
+  type todos {
+    title: String
+  }
+
+  type Query {
+    todos: [todos]
+    users: [User]
+  }
+  type User {
+    name: String!
+    password: String!
+    id: Int
+  }
+  type Mutation {
+      addToDo(title: String!, token: String!): [todos]
```

Comment:
Array of type `todos`? So each item is a todos? :smile: 

Why do you return an array for `addToDo` at all? Why not just return one (the added) todo?